ai16z/eliza
# docs: Update README.md

Relates to:

No issue; I created the pull request directly.

Risks:

Low

Background:

The starter script was not working because it did not include a cd command to navigate into the eliza-starter directory. This caused the script to fail when executed from the root of the project.

What does this PR do?

This PR updates the starter script by adding a cd command to ensure it correctly navigates into the eliza-starter directory before executing any subsequent steps.

What kind of change is this?
	•	Bug fixes (non-breaking change which fixes an issue)

Documentation changes needed?
	•	My changes do not require a change to the project documentation.

Testing:

Where should a reviewer start?

Start by reviewing the changes made to the starter script file.

Detailed testing steps:
	1.	Run the updated starter script from the root directory.
	2.	Confirm that the script correctly navigates into the eliza-starter directory and proceeds without errors.
